v0.3/tutorials/quickstart.eve

```diff
@@ -7,117 +7,95 @@ title: Quickstart
+## The Basics
+Let's start with the most basic example of Eve. Put this whole block, with the set of backticks (```) included, into the quickstart.eve file, then refresh your browser page where it's running.
 
-## Finding records in Eve
 
-Blocks search for records, and then bind or commit new records. Let's search for the `#greeting` we just committed, and then display it on the screen:
+```
+commit
+  [#ui/div text: "My Quickstart App"]
+```
 
-~~~eve
-search
-  [#greeting message]
+We want our webapp to be laid out nicely with a logo, a navigation bar, and a content area. To do that, add this whole block into the quickstart file right after the other block you already wrote, and refresh the browser page again.
 
-bind
-  [#ui/text text: message]
-~~~
 
-Searches find records that matches the supplied pattern, in this case records tagged greeting with a message attribute. You use these results to create new records. In this block, we create a `#ui/text` record for every message that is found (the UI library looks for records tagged `#ui/text` and draws them as DOM elements in the browser). If no records match the search, the block has no effect. Try adding another `#greeting` record in the first block to see what happens when more than one record matches the search. 
+```
+commit
+  [#ui/div #logo class:"qs-logo" sort: 1]
+  [#ui/div #nav-bar class:"qs-nav" sort: 2]
+  [#ui/div #contents class:"qs-contents" sort: 3]
+```
 
-Variables with the same name are equivalent within a block; because they have the same name, the `message` in `[#greeting message]` and `[#ui/text text: message]` are equivalent.
+The attribute `class` gets added to each div, which already has a little premade CSS to help see what we're doing. The logo is present, but next we need to add some pages to our nav bar and give each page some content.
 
-## Records update as data changes
 
-Blocks in Eve react automatically to changes in data. When a record changes, any bound records are automatically updated. We can see this in action by outputting the current time. First we create a timer that ticks once every second:
+## Pages for the Web App
+We can easily add a list of the pages we want for our app:
 
-~~~eve
+```
 commit
-  [#time #system/timer resolution: 1000]
-~~~
-
-Now we can search for the current time, and display it on the screen:
-
-~~~eve
-search
-  [#time seconds]
-
-bind 
-  [#ui/text text: seconds]
-~~~
+  [#page name: "Home" sort: 1]
+  [#page name: "Clock" sort: 2]
+  [#page name: "Canvas" sort: 3]
+```
 
-As the time changes, the output updates to reflect the current state of the `#time` record. Records can be committed instead of bound, but the behavior is a little different -- committed records persist until they are removed explicitly.
+These records alone don't add anything to the browser though. To actually make them appear, we need to write a block that first looks for our pages, then adds them into the browser.
```

Comment:
Mentioning this at the end has the same issue of users thinking they did something wrong and trying to debug before they continue reading. Perhaps we should move this note above the block?